### PR TITLE
Now uglifying/minifying only in Production

### PR DESCRIPTION
**Pull request by ilanusse**
## Summary
Gulp Build now uglifying/minifying only in Production environment.

## Test Plan
1) Run `npm install` and `bower update`.
2) Run `gulp development`.
3) Open the [server](http://localhost:3000). It should be working normally. The CSS and JS should NOT be minified/uglified.
4) Run `gulp production`.
5) Open the [server](http://localhost:3000). It should be working normally. The CSS and JS SHOULD be minified/uglified.

### CODE REVIEW

**Comment by ilanusse** [comment]
@sbalay

**Comment by ilanusse** [comment]
@sbalay Reminder!

File: README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ Run ```npm install``` from rootpath of the project.
 #### Gulp
 To start your app run ```gulp``` in the rootpath of the project. It is recommended to run gulp using the binary in the node modules folder ```./node_modules/.bin/gulp``` to avoid using an incorrect version of gulp that may have been installed globally. Consider adding ```alias gulp='node_modules/.bin/gulp'``` to your .bashrc o .bash_profile settings.
 
+If having problems with ```gulp watch```, run ```echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p```.
```
Comment (sbalay) [inline README.md]: Yo agregaría el src de donde sacamos esto, así el que quiere saber que hace el comando loco ese puede consultar la página que vimos

**Comment by sbalay** [comment]
:+1: 
Should we also minify the code of the stage env? I'm not sure. @ilanusse @framundo @guidomb

**Comment by framundo** [comment]
Yes, stage should replicate production as much as possible

**Comment by guidomb** [comment]
@sbalay Well at the build level production and stage should be same or as close as possible

**Comment by ilanusse** [comment]
@sbalay @framundo @guidomb I'm adding Staging support right now.

**Comment by sbalay** [comment]
@ilanusse Great! Feel free to merge this when you finish adding code minification to stage.

**Comment by ilanusse** [comment]
@sbalay Done. I can't merge, I'm not a collaborator.

**Pull request merged**